pools-drained/README.md

```diff
@@ -0,0 +1,33 @@
+# Pools Drained
+
+## Description
+
+This bot tries to detect Uniswap V2/V3 pools being drained.
+
+## Supported Chains
+
+- Ethereum
+- Optimism
+- Arbitrum
+- Polygon
+- All testnets related to above chains
+  
+> Note that Unisawp V2 is only deployed on Ethereum mainnet/testnets. The bot will work on
```

Comment:
```suggestion
> Note that Uniswap V2 is only deployed on Ethereum mainnet/testnets. The bot will work on
```

Maybe remove completely the "The bot..." sentence, as it may imply that the bot will never return a finding on other networks because they use a different factory when they don't use one at all.

Reply:
Or maybe change to:
The bot will work on other networks but will never return a finding if the Uniswap factories on that network doesn't correspond to the ones used in the bot.